api/types.go

```diff
@@ -302,7 +301,6 @@ type ListModelResponse struct {
 
 // ProcessModelResponse is a single model description in [ProcessResponse].
 type ProcessModelResponse struct {
-	Name      string       `json:"name"`
```

Comment:
Let's move the removal of redundant model Name responses to its own PR, it will be a breaking change so it should get proper visibility if/when we decide to merge it.